src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -921,6 +924,24 @@ internal void SuppressFastKillInHandleProcessExit()
                 ExecuteFastKillInProcessExit = false;
             }
 
+            internal IDictionary<GrainId, IGrainInfo> GetDirectory()
```

Comment:
I think these 2 methods (`GetDirectory` and `GetDirectoryForTypenamesContaining`) are not needed